import_ldraw.py

```diff
@@ -318,18 +290,37 @@ def parse(self, filename):
                                 x, y, z, a, b, c,
                                 d, e, f, g, h, i
                             ) = map(float, tmpdate[2:14])
-                            mat_new = self.mat * mathutils.Matrix((
-                                    (a, b, c, x),
-                                    (d, e, f, y),
-                                    (g, h, i, z),
-                                    (0, 0, 0, 1)
-                                ))
-
+                            # Reset orientation of top-level part, track original orientation
```

Comment:
Duplicate the `# TODO Check if isPart dependency can be avoided` comment here as well.